test/hotspot/jtreg/serviceability/jvmti/events/FramePop/framepop02/libframepop02.cpp

```diff
@@ -69,7 +69,7 @@ void print_current_time() {
 }
 
 static
-int isTestThread(JNIEnv *jni, jvmtiEnv *jvmti, jthread thr) {
+bool isTestThread(JNIEnv *jni, jvmtiEnv *jvmti, jthread thr) {
   jvmtiThreadInfo inf;
   const char* TEST_THREAD_NAME_BASE = "Test Thread";
   check_jvmti_status(jni, jvmti->GetThreadInfo(thr, &inf), "Error in GetThreadInfo.");
```

Comment:
In passing, should isTestThread deallocate inf.name to avoid leaking memory?

Reply:
Alan, thank you for looking at this fix.
Good suggestion.